### ISSUE TITLE
Accessibility: cannot tab to arrow buttons on small view

### ISSUE THREAD

**Issue by kburk1997** [Redocly/redoc#2284]
**Describe the bug**
Repro steps:
1. Go to redocly demo
2. Shrink browser window until arrow button appears
3. Tab through page

**Expected behavior**
The up/down arrow buttons are focusable.

**Actual behavior**
The up/down arrow buttons are not focusable.

**Additional context**
I filed a keyboard accessibility issue before - let me know if I should combine all accessibility issues I found.

Is there already a way to configure these buttons to be accessible?

**Comment by lornajane** [comment]
Thanks for logging these issues, we really appreciate it. No need to combine them, sharing what you spot works perfectly.
